src/components/example/EMultiSelect.tsx

```
@@ -0,0 +1,381 @@
+>(
+  (
+    {
+      options,
+      onValueChange,
+      variant,
+      defaultValue = [],
+      placeholder = "Select options",
+      animation = 0,
+      maxCount = 3,
+      modalPopover = false,
+      asChild = false,
```

ESLint

[error] 'asChild' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)